routing/features_road_graph.cpp

```diff
@@ -254,6 +254,40 @@ double FeaturesRoadGraph::GetSpeedKMPHFromFt(FeatureType const & ft) const
   return m_vehicleModel.GetSpeed(ft);
 }
 
+void FeaturesRoadGraph::ExtractRoadInfo(FeatureID const & featureId, FeatureType const & ft,
+                                        double speedKMPH, RoadInfo & ri) const
+{
+  Value const & value = LockFeatureMwm(featureId);
+  if (!value.IsAlive())
+  {
+    ASSERT(false, ());
```

Comment:
So, you just throw away a map if it does not have a section with altitudes, right?